crates/eww/src/app.rs

```diff
@@ -420,7 +510,7 @@ fn initialize_window(
 
     window.show_all();
 
-    Ok(EwwWindow { name: window_def.name.clone(), definition: window_def, gtk_window: window, scope_index: window_scope })
+    Ok(EwwWindow { instance_id: instance_id.to_string(), definition: window_def, gtk_window: window, scope_index: window_scope })
```

Comment:
eh, I'm not sure. Having it be the instance Id definitely makes sense, but then that only works when an instance id is specified. If otherwise it just defaults to the window name, I guess that should work

Reply:
Yup, currently the instance id will default to the config name if it is not specified